pilz_psenscan_test_area/execute_test.sh

```diff
@@ -5,20 +5,19 @@ log_ws_info()
   echo "ROS_PACKAGE_PATH=$ROS_PACKAGE_PATH"
 }
 
-launch_psen_scan_node()
+
+test_scan_compare()
 {
-  roslaunch $REPO_NAME psen_scan.launch sensor_ip:=192.168.0.10 host_ip:=192.168.0.20& 2>&1
-  PID=$!
-  echo "Started Process $PID"
+  TEST_FILE="hwtest_scan_compare.test"
+  TEST_ARGS="${TEST_DURATION+test_duration:=$TEST_DURATION} --text --results-base-dir $LOG_FOLDER_NAME"
```

Comment:
Not the best solution since `--text` corrupts the test result.

Reply:
https://github.com/ros/ros/issues/110

Reply:
I suggest we keep it for the moment. This has no effect on success/failure. But the output of `catkin_test_result` at the end can be incomplete.